### PR TITLE
Feature/models

### PR DESCRIPTION
**Pull request by SBK78MC**
Solved a name conflict and deleted the __pycache__
Furthermore, I added a class Camera.

### CODE REVIEW

File: backend/RayTracing/Classes/Models/Camera.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import math
+
+from RayTracing.Classes.Models.Vector import Vector
+
+
+class Camera():
+
+    angle = 0
+
+    def __init__(self, position=Vector(0,0,0), pointOfView=Vector(0,0,0), fov=0):
```
Comment (AlexDr7) [inline backend/RayTracing/Classes/Models/Camera.py]: Change default value of pointOfView to (0,0,1) and fov to 30 or 50

File: backend/RayTracing/Classes/RayTracer.py [hunk 2]
```diff
@@ -21,38 +24,72 @@ def __init__(self, imageplane=Imageplane(), mainscene=Scene(), camera=Camera()):
...
+                if ptcDotltp > 0:
+                    colorBrightness += light.getBrightness() * ptcDotltp / lightToPoint.calcLength()
+
+        initialColor = intersection.getObject().getColor().getArray()
+        finalColor = [initialColor[0]*colorBrightness, initialColor[1]*colorBrightness, initialColor[2]*colorBrightness]
+
+        return finalColor
 
 
 if __name__ == "__main__":
```
Comment (SBK78MC) [inline backend/RayTracing/Classes/RayTracer.py]: Please make sure you create a test for this, this keeps the Class "clean"

**Comment by SBK78MC** [comment]
Please make sure, that the tests run successful.

**Comment by AlexDr7** [comment]
Good work guys!!

**Pull request merged**